app/tests/src/route/test_index_route.py

```diff
@@ -0,0 +1,3 @@
+def test_get_index_200(client):
+    response = client.get("/")
+    assert response.status_code == 200
```

Comment:
Could you add a quick check that the HTML is right? At least that the response body has some of the text?